frame/staking/src/tests.rs

```diff
@@ -495,10 +487,10 @@ fn no_candidate_emergency_condition() {
 fn nominating_and_rewards_should_work() {
 	ExtBuilder::default()
 		.nominate(false)
-		.validator_pool(true)
+		.set_status(41, StakerStatus::Validator)
 		.build_and_execute(|| {
 			// initial validators -- everyone is actually even.
-			assert_eq_uvec!(validator_controllers(), vec![40, 30]);
+			assert_eq_uvec!(validator_controllers(), vec![40, 20]);
```

Comment:
shady change

Reply:
better to keep the same if its all the same

Reply:
Made the test more sensible, now we select 40 and 20 initially for a good reason.